internal/testrunner/system/system.go

```diff
@@ -0,0 +1,44 @@
+package system
+
+import (
+	"os"
+	"path"
+
+	"github.com/pkg/errors"
+)
+
+// Run runs the system tests for the given package.
+func Run(packageRootPath string) error {
+	systemTestsPath, err := findSystemTestsPath(packageRootPath)
```

Comment:
Just to confirm - there is one single directory for system test across entire package, not per every dataset?

EDIT:

I looked into the design paper and I think it mentions `test` directories in datasets instead. The reason why is it like that, is because you can enable different datasets in Elastic Agent and don't collect same logs in all test cases.

Reply:
Yes, you're right. It should be per dataset, not per package. Will fix, thanks for pointing this out!

Reply:
Fixed in 8275c27e0a25bb9e55cace984657391fa74fa272.